Makefile

```diff
@@ -44,6 +44,11 @@ BAZLISK_VERSION ?= 1.16.0
 BAZEL := $(shell command -v bazel 2> /dev/null)
 VENV = /dev/null # backward compatibility when no venv is built
 
+# Use x86_64 grpcurl v1.8.5 for Apple silicon chips
+ifeq ($(GRPCURL_OS)_$(MACHINE)_$(GRPCURL_VERSION), osx_arm64_1.8.5)
```

Comment:
Should we avoid hardcode `1.8.5` here?

Reply:
From v1.8.6 there is a dedicated grpcurl asset build for apple silicon chips. So this fix only applies to v1.8.5.

Reply:
We will be removing grpcurl (and lua-http) from dev dependency soon, in replace of a simple http2 + grpc tool in go from scratch. So consider this fix a temporary solution to unblock M1 mac users. I don't have strong opinion on the hardcoded version here.